examples/plot_knockoff_aggregation.py

```diff
@@ -1,209 +1,213 @@
 """
-Knockoff aggregation on simulated data
-======================================
-
-In this example, we show an example of variable selection using
-model-X Knockoffs introduced by :footcite:t:`candes2018panning`. A notable
-drawback of this procedure is the randomness associated with generating
-knockoff variables. This can result in fluctuations of the statistical power
-and false discovery proportion, and consequently, unstable inference.
-
-This example exhibits the two aggregation procedures described
-by :footcite:t:`pmlr-v119-nguyen20a` and :footcite:t:`Ren_2023` to derandomize
-inference.
+Knockoff aggregation
+====================
+
+The examples shows how to use aggregate model-X Knockoff selections to derandomize
+inference. The model-X Knockoff introduced by :footcite:t:`candes2018panning` allows
+for variable selection with statistical guarantees on the False Discovery Rate (FDR),
+
+.. math::
+    \text{FDR} = \mathbb{E}[\text{FDP}] = \mathbb{E} \left[ \frac{\hat{S} \cap \mathcal{H}_0}{|\hat{S}|} \right] ,
+
+where :math:`\hat{S}` is the set of selected variables and :math:`\mathcal{H}_0` is the
+set of null variables (i.e., variables with no effect on the response).
+A notable drawback of this procedure is the randomness associated with generating
+knockoff variables, :math:`\tilde{X}`. This can result in fluctuations of the statistical
+ power and false discovery proportion, and consequently, unstable inference.
+
+To mitigate this issue, several aggregation procedures have been proposed in the
+literature. :footcite:t:`pmlr-v119-nguyen20a` introduces a quantile aggregation
+procedure based on the p-values obtained from multiple independent runs of the
+knockoff filter. Or :footcite:t:`Ren_2023` proposes an aggregation procedure based on
+e-values. We illustrate both procedures in this example.
 """
 
-import matplotlib.pyplot as plt
+# %%
+# Generating data
+# ---------------
+# We use a simulated dataset where we know the ground truth to evaluate the
+# performance, in terms of statistical power and false discovery proportion, of the
+# different aggregation procedures.
+
 import numpy as np
-from joblib import Parallel, delayed
 
 from hidimstat._utils.scenario import multivariate_simulation
-from hidimstat.knockoffs import ModelXKnockoff
-from hidimstat.statistical_tools.multiple_testing import fdp_power
 
-# %%
-# Data simulation
-# ---------------
-# The comparison of the three methods relies on evaluating the
-# :term:`False Discovery Proportion <FDP>` (FDP) and statistical power. Assessing these
-# metrics requires knowledge of the actual data-generating process.
-# We therefore use simulated data with the following parameters:
-
-# number of repetitions of the methods
-runs = 20
-# Number of observations
-n_samples = 200
-# Number of variables
-n_features = 150
-# Correlation parameter
+n = 300
```

Comment:
If you can add the description of the parameter as before, it should be better.

Reply:
done